tests/test_tenant_mgt.py

```diff
@@ -211,60 +211,68 @@ def test_tenant_not_found(self, tenant_mgt_app):
 class TestCreateTenant:
 
     @pytest.mark.parametrize('display_name', [True, False, 1, 0, list(), tuple(), dict()])
-    def test_invalid_display_name(self, display_name, tenant_mgt_app):
+    def test_invalid_display_name_type(self, display_name, tenant_mgt_app):
         with pytest.raises(ValueError) as excinfo:
             tenant_mgt.create_tenant(display_name=display_name, app=tenant_mgt_app)
         assert str(excinfo.value).startswith('Invalid type for displayName')
 
+    @pytest.mark.parametrize('display_name', ['', 'foo', '1test', 'foo bar'])
```

Comment:
Maybe add a test for displayName over 20 characters?

Reply:
Good idea. Done.